source/server/active_listener_base.h

```diff
@@ -15,6 +15,8 @@ namespace Server {
   COUNTER(downstream_cx_overload_reject)                                                           \
   COUNTER(downstream_global_cx_overflow)                                                           \
   COUNTER(downstream_pre_cx_timeout)                                                               \
+  COUNTER(downstream_peek_remote_close)                                                            \
```

Comment:
I think this should be `downstream_pre_cx_remote_close` or `downstream_listener_filter_remote_close`. @mattklein123 any opinions on this name?

Reply:
at least I think `listener_filter` is much better than `peek`, thanks!